src/main/java/com/powsybl/openloadflow/network/LfGenerator.java

```diff
@@ -21,6 +25,10 @@
 
     boolean hasVoltageControl();
```

Comment:
I wonder if `hasVoltageControl` and `hasReactivePowerControl` (indeed it means remote control here) are still useful because it could be replaced by gettint the generator control type.